index.html

```diff
@@ -138,12 +143,13 @@ <h3 class="article__title">
             </div>
           </div>
 
-          <div class="article">
-            <img
+          <div class="article inArt" >
+            <div><img
               class="article__thumbnail"
-              src="edinburgh-damascus_thumbnail.png"
-              alt=""
-            />
+              src="images/edinburgh-damascus_thumbnail.png"
+              alt="a man riding a bike next a sign post of edinburgh-damascus "
+            /></div>
```

Comment:
Check the styling here and see if you can make it consistent with the rest of the tags

Reply:
I can't think of anything here. What's your suggest please?